supported-libraries-schema.json

```diff
@@ -0,0 +1,64 @@
+{
```

Comment:
What do we do for libraries whose configuration depends on the usage? For example, in netty channel handlers written by the user need to be annotated.

Reply:
Do you mean libraries/frameworks that generate configuration based on their annotations? I don't see a reason why those cannot be listed in here?